src/pages/Facility/settings/activityDefinition/ActivityDefinitionForm.tsx

```diff
@@ -211,6 +122,99 @@ function ActivityDefinitionFormContent({
   categorySlug?: string;
 }) {
   const { t } = useTranslation();
+
+  const formSchema = z.object({
+    title: z.string().min(1, t("title_is_required")),
+    slug_value: z
+      .string()
+      .min(1, t("field_required"))
+      .max(25, t("character_count_validation", { min: 1, max: 25 })),
+    description: z.string().min(1, t("field_required")),
+    usage: z.string().min(1, t("field_required")),
```

Comment:
```suggestion
    slug_value: z
      .string()
      .min(1, t("slug_is_required"))
      .max(25, t("character_count_validation", { min: 1, max: 25 })),
    description: z.string().min(1, t("description_is_required")),
    usage: z.string().min(1, t("usage_is_required")),
```

after this add the respective keys in en.json